sos/plugins/networking.py

```diff
@@ -208,6 +212,22 @@ def setup(self):
                         or line[:1].isspace():
                     continue
                 out_ns.append(line.partition(' ')[0])
+            # if namespaces defined, remove from out_ns not matching
+            # namespaces with regex
+            if self.get_option("namespaces"):
+                regex = '(?:%s)' % '|'.join(
+                      self.get_option("namespaces").split())
+                new_ns = []
+                for ns in out_ns:
```

Comment:
Can't be this `for ns in out_ns` cycle be merged into the previous one? I.e. why to build a list of namespaces to purge some items just afterwards?

Reply:
I have implemented new options into original loop of appending ns into out_ns. Please check new commit.